lib/alaveteli_text_masker.rb

```diff
@@ -139,11 +139,9 @@ def apply_pdf_masks(text, options = {})
           Rails.logger.warn "Unable to compress PDF; problem with your pdftk version?"
         end
 
-        if recompressed_text.blank?
-          text
-        else
-          recompressed_text
-        end
+        recompressed_text
```

Comment:
Fixed in https://github.com/mysociety/alaveteli/pull/3113/commits/791b676330316a78eca32287c1a11117b73794d5.